### PR TITLE
feat: add checkFreshness

### PR DESCRIPTION
**Pull request by Louis-Amas**

### CODE REVIEW

File: packages/bot-utils/src/checks.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import Mangrove from "@mangrovedao/mangrove.js";
+import { CommonLogger } from "./logging/coreLogger";
+
+const FRESHNESS_BLOCK_SECONDS = 60 * 5; // TODO: should be more precise
+const HOW_MANY_BLOCK_BACK = 10;
+
```
Comment (peterMangrove) [inline packages/bot-utils/src/checks.ts:6]: Could we make these to configurable?

**Review by peterMangrove — CHANGES_REQUESTED** [review]

File: packages/bot-utils/src/checks.ts (hunk 1 above)
Comment (Louis-Amas) [reply packages/bot-utils/src/checks.ts:6]: Yes we can but this is I hope a temporary measure, so do we want it ?

Comment (Louis-Amas) [reply packages/bot-utils/src/checks.ts:6]: I'm planning for a better version that requires a RES change I will update you shortly

Comment (Louis-Amas) [reply packages/bot-utils/src/checks.ts:6]: I was planning to add block timestamp inside the Block struct of reliable-events-subscriber but it's not possible as we don't have access to this info inside a multi call. I also started to look at what we currently we have in the mangrove.js config and we have `estimatedBlockTimeMs` we could use that and multiply it to a factor in config.

**Review by peterMangrove — APPROVED** [review]

**Pull request merged**